packages/g/gammaray/package.yml

```diff
@@ -0,0 +1,85 @@
+name       : gammaray
+version    : 3.0.0
+release    : 1
+source     :
+    - https://github.com/KDAB/GammaRay/releases/download/v3.0.0/gammaray-3.0.0.tar.gz: acd27dbbcbdf73fed497e0b5d6c477f2e11b59c48499752602677037dcd64ba5
+homepage   : https://www.kdab.com/development-resources/qt-tools/gammaray/
+license    : GPL-2.0-or-later
+component  :
+    - programming.tools
+    - probe-qt5 : programming.tools
+    - probe-qt5-devel : programming.devel
+summary    :
+    - GammaRay is a tool to poke around in a Qt-application
+    - probe-qt5 : Qt 5 probe for GammaRay
+    - probe-qt5-devel : Development files for gammaray-probe-qt5
+description:
+    - GammaRay is a software introspection tool for Qt applications developed by KDAB. Leveraging the QObject introspection mechanism it allows you to observe and manipulate your application at runtime. This works both locally on your workstation and remotely on an embedded target.
+    - probe-qt5 : Provides a Qt 5 probe for GammaRay that allows introspecting Qt 5 applications.
+    - probe-qt5-devel : Provides a Qt 5 probe for GammaRay that allows introspecting Qt 5 applications.
+clang      : yes
+optimize   : lto
+builddeps  :
+    - pkgconfig(Qt5Core)
+    - pkgconfig(Qt5Quick)
+    - pkgconfig(Qt5Script)
+    - pkgconfig(Qt5Scxml)
+    - pkgconfig(Qt5Svg)
+    - pkgconfig(Qt5WaylandCompositor)
+    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt63DAnimation)
+    - pkgconfig(Qt6Bluetooth)
+    - pkgconfig(Qt6Concurrent)
```

Comment:
Qt6Concurrent is in the Qt6Core package, so this can be dopped. You can probably just drop Qt6Core as well since it'll be pulled in transitively